frame/nfts/src/impl_nonfungibles.rs

```diff
@@ -135,6 +135,27 @@ impl<T: Config<I>, I: 'static> Inspect<<T as SystemConfig>::AccountId> for Palle
 	}
 }
 
+impl<T: Config<I>, I: 'static> InspectRole<<T as SystemConfig>::AccountId> for Pallet<T, I> {
+	/// Returns `true` if `who` is the issuer of the `collection`.
+	///
+	/// Default implementation is `false`, issuer is only added by `set_team`.
```

Comment:
I wouldn't put that note here since when you create a collection and pass the admin's address, that address would get all 3 roles by default.
The same applies to the freezer role.